### PR TITLE
 Integrate NVHTests and NVSTests in BlockchainSanity 

### PR DESCRIPTION
**Pull request by kushti**

### CODE REVIEW

File: testkit/src/main/scala/scorex/testkit/properties/NodeViewHolderTests.scala [hunk 1]
```diff
@@ -27,23 +27,29 @@ ST <: MinimalState[PM, ST],
...
+  class HolderFixture extends AkkaFixture {
     val (node, mod, s, h) = nodeViewHolder
   }
 
-  def createAkkaFixture(): Fixture = new HolderFixture
+  private def withFixture(testCode: HolderFixture => Any): Unit = {
+    val fixture = new HolderFixture
+    try {
+      testCode(fixture)
+    }
```
Comment (catena2w) [inline testkit/src/main/scala/scorex/testkit/properties/NodeViewHolderTests.scala]: We use one line codestyle, like
`} finally {`

**Comment by ceilican** [comment]
See https://github.com/ScorexFoundation/Scorex/pull/118 . I don't have push access to this repo. That's why I couldn't push the fix directly into this pull request.

**Comment by ceilican** [comment]
This is the test that failed:

```
[info] - NodeViewHolder: forking - switching *** FAILED ***
[info]   java.lang.NullPointerException:
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$39(NodeViewHolderTests.scala:239)
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$39$adapted(NodeViewHolderTests.scala:211)
[info]   at scorex.testkit.properties.NodeViewHolderTests.withFixture(NodeViewHolderTests.scala:47)
[info]   at scorex.testkit.properties.NodeViewHolderTests.$anonfun$$init$$38(NodeViewHolderTests.scala:211)
[info]   at scala.runtime.java8.JFunction0$mcV$sp.apply(JFunction0$mcV$sp.java:12)
[info]   at org.scalatest.OutcomeOf.outcomeOf(OutcomeOf.scala:85)
[info]   at org.scalatest.OutcomeOf.outcomeOf$(OutcomeOf.scala:83)
[info]   at org.scalatest.OutcomeOf$.outcomeOf(OutcomeOf.scala:104)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:22)
[info]   at org.scalatest.Transformer.apply(Transformer.scala:20)
[info]   ...
```

**Comment by ceilican** [comment]
Ok. I re-executed the failing build in Travis, and it succeeded this time...

**Comment by ceilican** [comment]
@catena2w I fixed the style, as you requested.

@kushti I fresolved the conflict.

@catena2w @kushti, I think this PR can be accepted now.

**Comment by ceilican** [comment]
I removed the obsolete failing test.

There is another test that sometimes fails, but this is not caused by this MR. The problem is discussed here (https://github.com/ScorexFoundation/Scorex/issues/119).

**Pull request merged**